apps/minifront/src/components/extension-not-connected.tsx

```diff
@@ -34,11 +35,12 @@ const handleErr = (e: unknown) => {
 
 const useExtConnector = () => {
   const [result, setResult] = useState<boolean>();
+  const navigate = useNavigate();
 
   const request = async () => {
     try {
       await requestPraxAccess();
```

Comment:
issue: after the update, reloading Minifront results in displaying the connection screen.

suggestion: add `isConnected()` check and, if connected, `navigate('/')`